backend/app/services/implementations/user_service.py

```diff
@@ -197,16 +198,13 @@ def create_user(self, user, auth_id=None, signup_method="PASSWORD"):
         new_user_dict = UserService.__user_to_serializable_dict_and_remove_auth_id(
             new_user
         )
-        new_user_dict["email"] = firebase_user.email
         kwargs = {
             "id": new_user_dict["id"],
-            "first_name": new_user_dict["info"]["primary_contact"]["name"],
-            "last_name": "",
-            "email": new_user_dict["email"],
-            "role": new_user_dict["info"]["role"],
+            "info": new_user_dict["info"],
         }
         return UserDTO(**kwargs)
 
+    # TODO: update this along with class UpdateUserByID
```

Comment:
Information about the specific mongoengine functions that are used can be found here: https://mongoengine-odm.readthedocs.io/apireference.html?highlight=modify#mongoengine.queryset.QuerySet.modify